File: 2037. Minimum Number of Moves to Seat Everyone/minMovesToSeat.js
/**
 * @param {number[]} seats
 * @param {number[]} students
 * @return {number}
 */
const minMovesToSeat = (seats, students) => {
  seats.sort((a, b) => a - b);
  students.sort((a, b) => a - b);

  let res = 0;
  for (let i = 0; i < students.length; i++) {
    res += Math.abs(students[i] - seats[i])
  }

  return res;
};

const seats = [4, 1, 5, 9];
const students = [1, 3, 2, 6];
console.log('minMovesToSeat: ', minMovesToSeat(seats, students));

File: 2037. Minimum Number of Moves to Seat Everyone/minMovesToSeat.py
from typing import List

class Solution:
  def minMovesToSeat(self, seats: List[int], students: List[int]) -> int:
    seats.sort()
    students.sort()
    
    res = 0
    for i in range(len(students)):
      res += abs(students[i] - seats[i])
      
    return res
  
  
seats = [4, 1, 5, 9]
students = [1, 3, 2, 6]
solution = Solution()
print('minMovesToSeat: ', solution.minMovesToSeat(seats, students))
